sky/cli.py

```diff
@@ -2011,6 +2011,13 @@ def queue(clusters: List[str], skip_finished: bool, all_users: bool):
     help=('Follow the logs of a job. '
           'If --no-follow is specified, print the log so far and exit. '
           '[default: --follow]'))
+@click.option(
+    '--number-of-lines',
```

Comment:
Shall we use '--tail' to be the same as Docker and K8s

Reply:
I think its a good idea to follow the convention of Docker and K8s, since @Michaelvll mentioned that `-n` has conflicted with existing naming convention of sky launch command